PROPOSALS.md

```diff
@@ -105,6 +105,13 @@ request feedback on feasibility before investing time and effort in the full pro
     * Unless your branch is particularly big, it's advisable to squash it into
       a single commit. At the moment GitLab does not offer "squash-and-merge"
       option in UI, so this has to be maintained manually.
+6. after the proposal is approved, coordinate with CLC and GHC team:
+    * The proposal needs to be labeled as "approved" and "awaits-merge"
+    * Add a comment to the GHC MR "Approved by CLC in [link to the vote outcome]" and get approval
+    * Double check that the changelog and the commit messages are in order
+    * Raise an issue on GHC GitLab to help GHC team with tracking
+    * MR needs to be assigned to Marge Bot, then wait until merged (done by either CLC or GHC devs)
+    * Remove proposal labels "approved" and "awaits-merge" and add "base-4.XX" (coordinate wiht GHC devs which base would be the first to release the change)
```

Comment:
No need to remove "approved" label though.

It's clear from the changelog which `base` will be the first to release the change; changes to `base` are not backported except special cirsumstances.